Topic 0/src/point2/BedRoom.java

```diff
@@ -0,0 +1,31 @@
+package point2;
+
+public class BedRoom implements Builder {
+	private Room room;
```

Comment:
[Medium] As it is not possible to change this object once it is created, you should declare it as final.

"A final variable can only be initialized once, either via an initializer or an assignment statement. It does not need to be initialized at the point of declaration: this is called a "blank final" variable. A blank final instance variable of a class must be definitely assigned in every constructor of the class in which it is declared."